src/main/kotlin/com/github/vol0n/utbotcppclion/server/GrpcServer.kt

```diff
@@ -175,6 +231,38 @@ class Server(private val port: Int) {
             logFinished(this::generatePredicateTests)
             return buildDummyTestsResponses(pathToGeneratedTestFile, generatedCode)
         }
+
+        override suspend fun heartbeat(request: Testgen.DummyRequest): Testgen.HeartbeatResponse {
+            return Testgen.HeartbeatResponse.newBuilder().setLinked(true).build()
+        }
+
+        override fun configureProject(request: Testgen.ProjectConfigRequest): Flow<Testgen.ProjectConfigResponse> {
```

Comment:
Is it used? Is it debug output? Can it be deleted?  
Why are this constants used? 
```
"one", "two", "Three", "four", "five" 
```
```
repeat(5) 
```
```
it.toDouble() / 4
```
```
delay(1000L)
```